terraform-infra-approvals/global.json

```diff
@@ -31,6 +31,7 @@
     {"source":  "git@github.com:hmcts/cnp-module-action-group?ref=master"},
     {"source":  "git@github.com:hmcts/cnp-module-application-insights?ref=master"},
     {"source":  "git@github.com:hmcts/cnp-module-app-service-plan?ref=master"},
-    {"source":  "git@github.com:hmcts/cnp-module-metric-alert?ref=master"}
+    {"source":  "git@github.com:hmcts/cnp-module-metric-alert?ref=master"},
+    {"source":  "git@github.com/hmcts/terraform-module-common-tags.git?ref=master"}
```

Comment:
there's no need for the common tags module on Jenkins, jenkins will provide it to you automatically via:

`var.common_tags`

Reply:
ah ok. will give it a go. thanks